### PR TITLE
Fix/scrollview get item layout

### PR DESCRIPTION
**Pull request by jan-kozinski**

### CODE REVIEW

**Review by asgalecki — APPROVED** [review]

File: src/screens/calendar/components/DayEvent.tsx [hunk 1]
```diff
@@ -17,8 +17,19 @@ export type DayOffEvent = {
 
 type DayEventProps = { event: DayOffEvent }
 
+const EVENT_VERTICAL_PADDING: keyof Theme['spacing'] = 's'
```
Comment (iceu-bb) [inline src/screens/calendar/components/DayEvent.tsx]: Can you make utils types for the theme, e.g 
type Spacing = keyof Theme['spacing']
type AvatarSize = keyof typeof avatarSizes

?

Comment (iceu-bb) [inline src/screens/calendar/components/DayEvent.tsx]: Can you make reusable utils types for the theme, e.g 
```ts
type Spacing = keyof Theme['spacing']
type AvatarSize = keyof typeof avatarSizes
```

?

File: src/screens/calendar/components/DayInfo.tsx [hunk 2]
```diff
@@ -1,27 +1,41 @@
...
 import { getDateWithMonthString, getDayName, isWeekend } from 'utils/dates'
-import { DayWeekend } from './DayWeekend'
-import { DayEvent } from './DayEvent'
+import { DayEvent, EVENT_HEIGHT } from './DayEvent'
 import { DayInfoProps } from '../../../types/DayInfoProps'
 
+const DAY_PADDING_VERTICAL = 'm'
+const OUTER_BOX_MARGIN = 's'
+const HEADING_TEXT_VARIANT = 'captionText'
+const FONT_OFFSET = 4
```
Comment (iceu-bb) [inline src/screens/calendar/components/DayInfo.tsx]: lack of typing :)

Comment (iceu-bb) [inline src/screens/calendar/components/DayInfo.tsx]: lack of typing :)

File: src/screens/calendar/components/EventsList.tsx [hunk 3]
```diff
@@ -23,17 +24,37 @@ export const EventsList = React.forwardRef<FlatList, EventsListProps>(({ days },
...
+export const getItemLayout = (data: DayInfoProps[] | null | undefined, index: number) => {
+  if (!data)
+    return {
+      length: DAY_ITEM_HEIGHT,
+      offset: DAY_ITEM_HEIGHT * index,
+      index,
+    }
+  let prevEventsCount = 0
+  for (let i = 0; i < index; i++) {
+    prevEventsCount += data[i]?.events?.length ? data[i].events?.length || 0 : 0
```
Comment (iceu-bb) [inline src/screens/calendar/components/EventsList.tsx]:
```suggestion
    prevEventsCount += data[i].events?.length || 0 
```

Comment (iceu-bb) [inline src/screens/calendar/components/EventsList.tsx]:
```suggestion
    prevEventsCount += data[i].events?.length || 0 
```

**Review by iceu-bb — APPROVED** [review]

File: src/screens/calendar/components/DayEvent.tsx (hunk 1 above)
Comment (jan-kozinski) [reply src/screens/calendar/components/DayEvent.tsx]: yeah, sure

**Pull request merged**